### DISCUSSION TITLE
How to update the new password of the One Drive?

### DISCUSSION THREAD

**Discussion by rejusam** [abraunegg/onedrive#1602] [Q&A] [answered]
I recently updated my account password. How to add this new password to the onedrive client?

**Comment by abraunegg** [comment, marked as answer]
@rejusam 
This is documented in the usage document: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#handling-a-onedrive-account-password-change

**Reply by rejusam** [reply]
Thank you... :-)
